openml/tasks/task.py

```diff
@@ -177,6 +177,20 @@ def __init__(self, task_id, task_type_id, task_type, data_set_id,
         )
         self.number_of_clusters = number_of_clusters
 
+    def get_X(self, dataset_format='array'):
+        """Get data associated with the current task.
+
```

Comment:
type hint, doc string with legal values for `dataset_format` parameter.

Reply:
Done.